dockstore-webservice/src/main/java/io/dockstore/webservice/core/Organization.java

```diff
@@ -100,6 +107,13 @@
     @ApiModelProperty(value = "Display name for an organization (Ex. Ontario Institute for Cancer Research). Not used for links.", position = 9)
     private String displayName;
 
+    @ManyToMany(fetch = FetchType.LAZY, cascade = CascadeType.MERGE)
+    @JoinTable(name = "starred_organizations", inverseJoinColumns = @JoinColumn(name = "userid", nullable = false, updatable = false, referencedColumnName = "id"), joinColumns = @JoinColumn(name = "organizationid", nullable = false, updatable = false, referencedColumnName = "id"))
+    @ApiModelProperty(value = "This indicates the users that have starred this organization", required = false, position = 10)
```

Comment:
Probably beyond the scope of this, but I wonder if we shouldn't be setting `accessMode = ApiModelProperty.AccessMode.READ_ONLY` on properties like this?

Reply:
Its worth a thought. 
I wonder if it improves the codegen/makes it more intuitive